cmd/kyma/test/run/cmd.go

```diff
@@ -42,17 +50,25 @@ To execute all test defintions, run ` + "`kyma test run -n example-test`" + `.
 	}
 
 	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Specifies the name of the new test suite. If you don't specify the value for the `-n` flag, the name of the test suite will be autogenerated.")
-	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, "Defines how many times every test should be executed. \"count\" and \"max-retries\" flags are mutually exclusive.")
-	cobraCmd.Flags().Int64VarP(&o.MaxRetries, "max-retries", "", 0, "Defines how many times a given test is retried when it fails. A suite is marked with a \"succeeded\" status even if some tests failed at first and then finally succeeded. The default value of 0 means that there are no retries of a given test.")
+	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, `Defines how many times every test should be executed. "count" and "max-retries" flags are mutually exclusive.`)
+	cobraCmd.Flags().Int64VarP(&o.MaxRetries, "max-retries", "", 0, `Defines how many times a given test is retried when it fails. A suite is marked with a "succeeded" status even if some tests failed at first and then finally succeeded. The default value of 0 means that there are no retries of a given test.`)
 	cobraCmd.Flags().Int64VarP(&o.Concurrency, "concurrency", "", 1, "Specifies the number of tests to be executed in parallel.")
+	cobraCmd.Flags().DurationVar(&o.Timeout, "timeout", 10*time.Minute, `Maximum time during which the test suite is being watched. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
```

Comment:
A typical kyma test suite will take around 30minutes I guess..
usually the watch gets interrupted by finishing the test suite. The timeout is only for the case of hanging tests and not having a hanging CI. To detect hanging tests you need to wait much more than the regular suite execution duration.
So shouldn't be the default something around 1 hour?

Or maybe better to have no default at all, so by default it runs forever. For local execution it is probably what you want. For CI situation you will specify a timeout always.

Reply:
I've checked also the `kubectl` approach and basically it's messy, they have such policies for timeout:

- `"The length of time to wait before giving up on a delete, zero means determine a timeout from the size of the object"`
- "The length of time to wait before giving up, zero means infinite"
- "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week."



but IMO adding opt that default it runs forever is the best option, so I changed and now the default value is zero which means infinite timeout.